docs: add copy option return_failed_only. (#13140)
datafuselabs/databend: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

| ON_ERROR              | Decides how to handle a file that contains errors: 'continue' to skip and proceed, 'abort' to terminate on error, 'abort_N' to terminate when errors ≥ N. Default is 'abort'. Note: 'abort_N' not available for Parquet files.                                                                                                                                                                                   | Optional |
| MAX_FILES             | Sets the maximum number of files to load that have not been loaded already. The value can be set up to 500; any value greater than 500 will be treated as 500.                                                                                                                                                                                                                                                   | Optional |

> | RETURN_FAILED_ONLY    | Boolean that specifies whether to return only files that have failed to load in the statement result. Default: `False`.                                                                                                                                                                                                                                                                                          | Optional |

Reviewer: ```suggestion
| RETURN_FAILED_ONLY    | When set to 'True,' only files that failed to load will be returned in the output. Default: `False`.                                                                                                                                                                                                                                                                                          | Optional |
```
